app/routers/event.py

```diff
@@ -79,7 +79,7 @@ async def create_new_event(request: Request,
 
     messages = get_messages(session, event, uninvited_contacts)
     return RedirectResponse(router.url_path_for('eventview', event_id=event.id)
-                            + f'messages={"---".join(messages)}',
+                            + f'?messages={"---".join(messages)}',
```

Comment:
Why the ? sign?

Reply:
This is the way to pass additional query parameters in `get` requests ([Take a look at the syntax portion of this wikipedia article about URLs](https://en.wikipedia.org/wiki/URL)). The redirection didn't work without this addition.

Reply:
ok, thanks for the info!